src/middleware/middleware.ts

```diff
@@ -4,11 +4,15 @@ import { getPayload } from "./get-payload";
 import { verifyAndReceive } from "./verify-and-receive";
 import { debug } from "debug";
 import { IncomingMessage, ServerResponse } from "http";
-import { State, OctokitError, WebhookEventHandlerError } from "../types";
+import { State, WebhookEventHandlerError } from "../types";
 import { EventNames } from "../generated/event-names";
-import AggregateError from "aggregate-error";
+import consoleLogLevel from "console-log-level";
 
 const debugWebhooks = debug("webhooks:receiver");
+const logWebhooks = consoleLogLevel({
+  prefix: "webhooks:receiver",
+  level: "info",
+});
```

Comment:
Not sure if this is the approach you were looking for @gr2m , when you have time I would like to hear your feedback

Reply:
I would not use it, it increases the bundle size, and I want `@octokit/webhooks` to work well in contraint environments such as browsers and Cloudflare Workers in future.

We can recommend to use it to users as we do in `@octokit/core`: https://github.com/octokit/core.js#logging

Reply:
Does this issue satisfy your request? If so, can we close this PR and associated issue? https://github.com/octokit/webhooks.js/issues/476